### PR TITLE
Add clang-format support

### PR DESCRIPTION
**Pull request by PSUdaemon**

### CODE REVIEW

**Comment by zwoop** [comment]
Looks great.

**Comment by linuxwolf** [comment]
@PSUdaemon ... please add yourself to AUTHORS

Also, would like a note in the README about any requirements regarding clang-format (version, etc).

**Comment by linuxwolf** [comment]
@zwoop ... please add yourself to AUTHORS

File: README.md [hunk 1]
```diff
@@ -86,3 +86,12 @@ This has been seen on Mac OSX 10.9 when openssl 1.0.1h or newer has been install
...
     ./configure --with-jansson=/usr/local/opt/jansson
+
+## Contributing ##
+
+### Before Submitting PR ###
+
+* Run `make clang-format`
+* Run `make test`
+
+*NOTE* You must use clang-format 3.9.0. You can download binaries from [here](https://bintray.com/apache/trafficserver/clang-format-tools).
```
Comment (linuxwolf) [inline README.md]: near the top of this file is a "Prerequisites" section where this should go

**Pull request merged**